data/human/hails.txt

```diff
@@ -687,6 +713,8 @@ phrase "friendly civilian"
 		"Want my opinion? You should"
 		"Be like the best ship captains and"
 		"Don't forget to"
```

Comment:
This can lead to the combination "Don't forget to don't bother with shield generators." This doesn't sound something that somebody would actually say.